### PR TITLE
Update Split_large_file.md

### PR DESCRIPTION
**Pull request by eporetsky**
Currently, the last batch is not yielded by the batch_iterator function. After the for-loop is done iterating, yield the last batch if it has any entries. Hope that I got it right!

### CODE REVIEW

File: wiki/Split_large_file.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ def batch_iterator(iterator, batch_size):
         if len(batch) == batch_size:
             yield batch
             batch = []
+    if len(batch) > 0:
+        yield batch
 ```
```
Comment (peterjc) [inline wiki/Split_large_file.md:51]: Well spotted, perhaps it was only ever tested on examples with no remainder (e.g. 50 entries in batches of 10).

Style wise I'd suggest ``if batch:`` but the fix looks good.

Comment (eporetsky) [reply wiki/Split_large_file.md:51]: > Well spotted, perhaps it was only ever tested on examples with no remainder (e.g. 50 entries in batches of 10).
> 
> Style wise I'd suggest `if batch:` but the fix looks good.

Makes sense. I had a remainder in my non-perfect fasta. 
And `if batch:` does look better.
Thanks!

Comment (peterjc) [reply wiki/Split_large_file.md:51]: If you make that minor improvement, I'll merge this.

**Comment by peterjc** [comment]
Thank you!

**Pull request merged**